res/css/views/dialogs/_SettingsDialog.scss

```diff
@@ -14,37 +14,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.mx_SettingsDialog {
-    .mx_Dialog {
-        height: 80%;
-        border-radius: 4px;
-        padding-top: 0;
-        padding-right: 30px;
-        padding-left: 0;
-
-        .mx_TabbedView {
-            top: 65px;
-        }
-
-        .mx_TabbedView .mx_SettingsTab {
-            box-sizing: border-box;
-            min-width: 580px;
-            padding-right: 100px;
-
-            // Put some padding on the bottom to avoid the settings tab from
-            // colliding harshly with the dialog when scrolled down.
-            padding-bottom: 100px;
-        }
-
-        .mx_Dialog_title {
-            text-align: center;
-            margin-top: 16px;
-            margin-bottom: 24px;
-        }
-
-        .mx_Dialog_fixedWidth {
-            max-width: 1000px;
-            width: 90vw;
-        }
+// Not actually a component but things shared by settings components
+.mx_UserSettingsDialog, .mx_RoomSettingsDialog {
+    .mx_TabbedView {
+        top: 65px;
     }
+
+    .mx_TabbedView .mx_SettingsTab {
+        box-sizing: border-box;
+        min-width: 580px;
+        padding-right: 100px;
+
+        // Put some padding on the bottom to avoid the settings tab from
+        // colliding harshly with the dialog when scrolled down.
+        padding-bottom: 100px;
+    }
+
+    .mx_Dialog_title {
+        text-align: center;
+        margin-bottom: 24px;
+    }
+
+    max-width: 1000px;
+    width: 90vw;
+    // set the height too since tabbed view scrolls itself.
+    height: 80vh;
```

Comment:
Personal preference is to have these at the top rather than bottom, but I'm also not sure what our conventions or guidelines say, or if it even matters.

Reply:
actually yes, that's probably what I'd expect too